### ISSUE TITLE
Demo video prep checklist

### ISSUE THREAD

**Issue by kgodey** [centerofci/mathesar#2079]
Here's the list of things we need before we can do screen recording for the demo video:

- [x] Temporary `http://library.mathesar.org/` server
  - [x] Set up server in GCP - @mathemancer 
  - [x] Set up DNS - @kgodey 
  - [x] Set up deployment in Ansible repo - @mathemancer 
  - [x] Set up SQL access for video production team  - @mathemancer 
- [x] Data sets
  - [x] A pair of denormalized books and patrons TSVs (with the real book data). - @mathemancer 
  - [x] An SQL file defining the final data schema with data in all tables _except_ checkouts. The checkouts table would be created (with appropriate constraints), but empty. - @mathemancer 
  - [x] An SQL file that loads checkout data into the checkouts table. - @mathemancer 
  - [x] Fix erroneous record and duplicate publisher - @mathemancer 
- [x] SQL
  - [x] Creating a record using the email data type and showing validation  - @mathemancer 
  - [x] Setting up an exclusion constraint (or some other "set up something in the SQL that Mathesar doesn't have" thing)  - @mathemancer 
- [x] Documentation 
  - [x] Write up instructions on how to run the appropriate SQL for the demo video team (including SSH info)  - @mathemancer 
  - [x] Augment instructions to include what needs to be configured before each step in script - @kgodey 
- [x] Pick voice actor - @kgodey 
- [x] Annotate script with feedback - @kgodey 
- [x] Send feedback to video production team - @kgodey 
- [x] Replace references in video script with actual author / book names - @kgodey 
- [x] Walk through script and file bugs if needed - @kgodey
- [x] Ensure must-haves below are done and deploy latest code - @kgodey 
- [x] Schedule walkthrough with video production team - @kgodey 
- [x] Send documentation and data sets to video production team - @kgodey 

### Must-have
- [x] #1644
- [x] #2092
- [x] #1667 - @seancolsen
- [x] #1622 - @rajatvijay
- [x] #1959
- [x] #2010 - @pavish
- [x] #1678 - @rajatvijay
- [x] #2113 - @pavish
- [x] #2119 - @seancolsen
- [x] #2021 - @pavish
- [x] #2163 - @pavish 

### Must-have - backend
- [x] #2143 
- [x] ~#2144~ Worked around this by updating data to remove ID in https://github.com/centerofci/mathesar-data-playground/pull/16
- [x] Column has wrong name – see comment in https://github.com/centerofci/mathesar-data-playground/pull/16
- [x] https://github.com/centerofci/mathesar/issues/2145 - @dmos62
- [x] #2152 - @seancolsen and @silentninja 

### Nice-to-have - frontend
- [x] #1921 - @rajatvijay
- [x] #2051 - @rajatvijay
- [ ] #2011 - @pavish
- [ ] #2104 - @seancolsen
- [x] #2025 - @rajatvijay
- [ ] #2105 - @ghislaineguerin
- [x] #2156 - @seancolsen 

### Nice-to-have - backend
- [ ] https://github.com/centerofci/mathesar/issues/2153 - @Anish9901

**Comment by kgodey** [comment]
I did a run through of the demo script and everything looks great!
